go.mod

```diff
@@ -1,54 +1,54 @@
+	k8s.io/apiserver => k8s.io/apiserver v0.24.5
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.24.5
 	k8s.io/client-go => github.com/rancher/client-go v1.24.0-rancher1
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.24.2
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.24.2
-	k8s.io/code-generator => k8s.io/code-generator v0.24.2
-	k8s.io/component-base => k8s.io/component-base v0.24.2
-	k8s.io/component-helpers => k8s.io/component-helpers v0.24.2
-	k8s.io/controller-manager => k8s.io/controller-manager v0.24.2
-	k8s.io/cri-api => k8s.io/cri-api v0.24.2
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.24.2
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.24.2
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.24.2
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.24.2
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.24.2
-	k8s.io/kubectl => k8s.io/kubectl v0.24.2
-	k8s.io/kubelet => k8s.io/kubelet v0.24.2
-	k8s.io/kubernetes => k8s.io/kubernetes v1.24.2
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.24.2
-	k8s.io/metrics => k8s.io/metrics v0.24.2
-	k8s.io/mount-utils => k8s.io/mount-utils v0.24.2
-	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.24.2
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.24.2
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.24.5
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.24.5
+	k8s.io/code-generator => k8s.io/code-generator v0.24.5
+	k8s.io/component-base => k8s.io/component-base v0.24.5
+	k8s.io/component-helpers => k8s.io/component-helpers v0.24.5
+	k8s.io/controller-manager => k8s.io/controller-manager v0.24.5
+	k8s.io/cri-api => k8s.io/cri-api v0.24.5
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.24.5
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.24.5
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.24.5
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.24.5
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.24.5
+	k8s.io/kubectl => k8s.io/kubectl v0.24.5
+	k8s.io/kubelet => k8s.io/kubelet v0.24.5
+	k8s.io/kubernetes => k8s.io/kubernetes v1.24.5
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.24.5
+	k8s.io/metrics => k8s.io/metrics v0.24.5
+	k8s.io/mount-utils => k8s.io/mount-utils v0.24.5
+	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.24.5
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.24.5
 )
 
 require (
 	github.com/golang/mock v1.6.0
 	github.com/gorilla/mux v1.8.0
 	github.com/rancher/dynamiclistener v0.3.2-0.20211221223804-148d38076d3e
-	github.com/rancher/lasso v0.0.0-20220214201210-e50bfe3e2871
+	github.com/rancher/lasso v0.0.0-20220627205005-00d9c8e9dda6
 	github.com/rancher/lasso/controller-runtime v0.0.0-20220110204826-724e3e57c47d
-	github.com/rancher/rancher/pkg/apis v0.0.0-20210628154046-7a2fc74f9598
+	github.com/rancher/rancher/pkg/apis v0.0.0-20221103011339-cbbef69d4c2e
```

Comment:
Thank you for updating this very out of date dependency. I think we should add this to the release steps for the adapter - i.e. update this value when RCs start being cut so that we are in line with rancher as much as possible (ideally we would have exactly what ships with the version that the webhook is for but that probably won't happen).